CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
     - "Transferred to another public health facility"
     - "Moved to private practitioner"
 - Build only the `PRODUCTION` release APK as part of the CI checks
+- Migrate `PatientSummaryScreen` to a fragment
```

Comment:
Can we put this in a different section since it's not part of the new `DEMO` release?

Reply:
I am waiting for the CHANGELOG PR to be merged in so that I can fix this when rebasing.